recipes/gffutils/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "0.9" %}
+{% set version = "0.10.1" %}
 
 package:
   name: gffutils
   version: {{ version }}
 
 build:
   noarch: python
-  number: 1
+  number: 0
 
 source:
-  url: https://pypi.python.org/packages/b4/ea/54ca403a8d471849606fb432b9afcf73c8c5105ea2dd87b8d38bd5217c5a/gffutils-{{ version }}.tar.gz
-  sha256: e1b0bf2b422ec3491be13d77bf7baf71d274abd844e8f8b6c1217a95b236218b
+  url: https://files.pythonhosted.org/packages/87/d6/f893f14fd6cf1c9e758259f6e0f9ba1ae03370988d8f1a3ea8d790ca5b06/gffutils-0.10.1.tar.gz
```

Comment:
The url should probably be `https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`.